recipe/meta.yaml

```
@@ -24,7 +24,8 @@ requirements:
   run:
     - python
     - python-dateutil >=2.4
-    - typing-extensions >=3.10.0.1 # [py<38]
+    - text-unidecode ==1.3
+    - typing-extensions >=3.10.0.2
```

```suggestion
    - typing-extensions >=3.10.0.2  # [py<38]
```
https://github.com/joke2k/faker/blob/v10.0.0/setup.py#L74